service.go

```diff
@@ -161,7 +161,7 @@ func (s *Service) name() string {
 
 	w.WriteString(s.Name)
 
-	if s.Revision != "" {
+	if s.Revision != "" && len(s.Revision) > 5  {
```

Comment:
This can just be a single check of len() > 5

Reply:
Look at the galaxy brain of this one! Good move.

Reply:
A very good point. I will implement